tests/functions.sh

```diff
@@ -21,7 +21,7 @@ RETURN_CODE=0
 function return_result {
     clean_all
     if [ $RETURN_CODE == 0 ]; then
-        echo $CUR_TEST 'SUCESS !!!'
```

Comment:
nice, can you move this to another commit please

Reply:
de suite!